src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -656,3 +658,20 @@ message LongRunningConfigProto {
   // Total polling timeout.
   uint64 total_poll_timeout_millis = 8;
 }
+
+message OutputSpec {
+  message LoopStatement {
+    string collection = 1;
+    string variable = 2;
+    repeated OutputSpec body = 3;
+  }
+
+  // Exactly one of the following should be set.
+
+  LoopStatement loop = 1;
+
+  // example ["%s=%s", thing.variable, thing.value]
+  // field name is singular because the array
+  // represents one print statement.
+  repeated string print = 2;
```

Comment:
`print` is a reserved word in numerous languages. It is also a verb; proto fields should generally be nouns.

I can not actually determine what is intended -- does it represent a print _statement_ or does it represent the _string to be printed_?

Reply:
@lukesneeringer See my previous comment. I'm the one who suggested the repeated string `print` because it would make the input yaml more readable, comprehensible, and compact, like so:

```
- print:
  - "Person %s lives at %s\n"
  - response.name
  - response.address
```

After all, the output_spec field is a description of the code to output, so I'd like it be as easy for humans as possible. This seems very readable, as it's the name of the statement (indeed, a verb and a common reserved word: that is on purpose) and then the list args that get used in printf-style functions in many languages.

Let me also say that generally I favor good encapsulation and reusable messages so my first inclination (eg in the design doc) was indeed to to have a `PrintStatement`. But since a big part of the purpose of this proto is to generate the yaml schema used by API teams to define their samples, I deem that the usability gains are worth it in this case. I'm open to convincing counterarguments.